sample/src/main/java/com/airbnb/airmapview/sample/MainActivity.java

```diff
@@ -86,6 +86,25 @@
     return super.onOptionsItemSelected(item);
   }
 
+  private AirMapInterface createMapWithTouchDisabled(AirMapViewTypes type) {
+    if (type == AirMapViewTypes.NATIVE) {
+      AirGoogleMapOptions options = new AirGoogleMapOptions(new GoogleMapOptions());
+      options.setTouchEnabled(false);
+      AirMapInterface airMapInterface = mapViewBuilder.builder(type).withOptions(options).build();
+      return airMapInterface;
+    }
+
+    // case where it's not a native map, doesn't use AirGoogleMapOptions
```

Comment:
I feel like adding this functionality to the sample app in the menu isn't the best way to do it. We should probably create a separate configuration screen, from which users can tweak all of the available settings for any type of map. 

If you remove the changes from the sample app, it looks good to merge, then I can integrate it into the sample app in a separate configuration screen.